doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,876 @@
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.9.0`_ for the changes required or recommended when migrating your application from |NCS| v2.8.0 to |NCS| v2.9.0.
+
+.. _ncs_release_notes_290_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE, OS, and tool support
+=========================
+
+* Added Nordic Thingy:91 X to the list of devices supported by the `Quick Start app`_ on the :ref:`gsg_guides` page.
+
+* Updated the deprecation notes for `nRF Command Line Tools`_ added in the previous release.
+  The notes now clearly state the tools will be archived: no updates will be made to the software, but it will still be available for download.
+
+Board support
+=============
+
+* Updated various tests and samples to use Zephyr's :ref:`native_sim<zephyr:native_sim>` instead of Zephyr's :ref:`native_posix<zephyr:native_posix>` for :ref:`running_unit_tests`.
+  This mirrors the deprecation of ``native_posix`` in Zephyr.
+  Support for ``native_posix`` will be removed in Zephyr with the v4.2 release.
+  In the |NCS|, it will be removed once Zephyr v4.2 is upmerged to sdk-nrf.
+
+Build and configuration system
+==============================
+
+* Fixed the issue in the ``nordic-bt-rpc`` snippet where an invalid memory map was created for nRF54H20 devices, which resulted in a runtime failure.
+
+Bootloaders and DFU
+===================
+
+* Updated the allowed offset for :ref:`doc_fw_info` in the :ref:`bootloader`.
+  It can now be placed at offset ``0x600``.
+  However, you cannot use it for any applications with |NSIB| compiled before this change.
+
+Developing with nRF91 Series
+============================
+
+* Moved the Thingy:91 and Thingy:91 X guides to new sections, :ref:`thingy91_ug_intro` and :ref:`ug_thingy91x` respectively, under :ref:`ug_app_dev`.
+
+Developing with nRF70 Series
+============================
+
+|no_changes_yet_note|
+
+Working with nRF54H Series
+==========================
+
+|no_changes_yet_note|
+
+Developing with nRF54L Series
+=============================
+
+* Added the :ref:`nRF54l_signing_app_with_flpr_payload` page that includes instructions for building separate applications, merging them, and signing them for MCUboot.
+
+Developing with nRF53 Series
+============================
+
+* Moved the Thingy:53 to a new section, :ref:`ug_thingy53`, under :ref:`ug_app_dev`.
+
+Developing with nRF52 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with Front-End Modules
+=================================
+
+* Deprecated the explicit use of ``-DSHIELD=nrf21540ek_fwd`` for boards with ``nrf5340/cpuapp`` qualifiers when the nRF21540 EK shield is used.
+  The build system uses an appropriate overlay file for each core, relying only on the ``-DSHIELD=nrf21540ek`` parameter.
+
+Developing with PMICs
+=====================
+
+|no_changes_yet_note|
+
+Security
+========
+
+|no_changes_yet_note|
```

Comment:
* Extended ``west ncs-provision`` command, so different key lifetime policies can be selected.